editor/components/post-featured-image/index.js

```diff
@@ -24,6 +24,9 @@ const DEFAULT_REMOVE_FEATURE_IMAGE_LABEL = __( 'Remove featured image' );
 
 function PostFeaturedImage( { featuredImageId, onUpdateImage, onRemoveImage, media, postType } ) {
 	const postLabel = get( postType, [ 'labels' ], {} );
+	const featuredImageAltText = ( altText ) => {
+		return altText ? ' Current image: ' + altText : '';
```

Comment:
This text needs to be wrapped with the translation helper.

Reply:
@gziolo  I think I've fixed this, can you check if I've missed something